eslintrc.json

```diff
@@ -1,169 +1,156 @@
 {
-  "ecmaFeatures": {
-    "modules": true
-  },
+  "parser": "babel-eslint",
 
   "env": {
-    "browser": true,
-    "es6": true,
-    "node": true
-  },
-
-  "globals": {
-    "document": false,
-    "escape": false,
-    "navigator": false,
-    "unescape": false,
-    "window": false
+    "commonjs": true,
```

Comment:
though, we don't technically use commonjs, is there any benefit to having it enabled?